nncf/quantization/algorithms/bias_correction/torch_fx_backend.py

```diff
@@ -0,0 +1,121 @@
+from nncf.torch.tensor_statistics.collectors import get_mean_statistic_collector
+from nncf.torch.tensor_statistics.collectors import get_raw_stat_collector
+
+
+class FXBiasCorrectionAlgoBackend(BiasCorrectionAlgoBackend):
+
+    @staticmethod
+    def target_point(target_type: TargetType, target_node_name: str, port_id: int) -> PTTargetPoint:
+        return get_target_point(target_type, target_node_name, port_id)
+
+    @staticmethod
+    def create_bias_correction_command(
+        node: NNCFNode, bias_value: Tensor, nncf_graph: NNCFGraph
+    ) -> FXApplyTransformationCommand:
+        return FXApplyTransformationCommand(bias_update_transformation_builder(node, bias_value.data))
+
+    @staticmethod
+    def model_extraction_command(
+        input_ids: Set[Tuple[str, int]], output_ids: Set[Tuple[str, int]]
+    ) -> PTModelExtractionCommand:
+        return PTModelExtractionCommand([inp_id[0] for inp_id in input_ids], [out_id[0] for out_id in output_ids])
+
+    @staticmethod
+    def output_insertion_command(nncf_graph: NNCFGraph, target_point: PTTargetPoint) -> FXApplyTransformationCommand:
+        return FXApplyTransformationCommand(output_insertion_transformation_builder(target_point))
+
+    @staticmethod
+    def mean_statistic_collector(
+        channel_axis: int,
+        inplace: bool,
+        num_samples: Optional[int] = None,
+        window_size: Optional[int] = None,
+    ) -> TensorCollector:
+        return get_mean_statistic_collector(num_samples, channel_axis, window_size)
+
+    @staticmethod
+    def raw_statistic_collector(num_samples: Optional[int] = None) -> TensorCollector:
+        return get_raw_stat_collector(num_samples)
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: int) -> Tensor:
+        return Tensor(raw_data[output_name])
+
+    @staticmethod
+    def get_activation_port_id(node: NNCFNode, nncf_graph: NNCFGraph) -> int:
+        return 0
+
+    @staticmethod
+    def get_bias_value(node: NNCFNode, model: torch.fx.GraphModule, nncf_graph: NNCFGraph) -> Tensor:
+        return Tensor(get_bias_value(node, nncf_graph, model))
+
+    @staticmethod
+    def get_input_name(model: torch.fx.GraphModule, node_name: str, input_port_id: int) -> str:
+        graph_node = get_graph_node_by_name(model.graph, node_name)
+        return graph_node.all_input_nodes[input_port_id].name
+
+    @staticmethod
+    def get_output_name(model: torch.fx.GraphModule, node_name: str, output_port_id: int) -> int:
+        graph_node = get_graph_node_by_name(model.graph, node_name)
+        nodes = list(graph_node.users)
+        while nodes:
+            node = nodes.pop()
+            if node.op == "call_function" and node.target == torch.ops.aten.clone.default:
+                nodes = list(node.users)
+                graph_node = node
+            elif node.op == "output":
+                return node.all_input_nodes.index(graph_node)
```

Comment:
Just for interest.
As i see output node in FXGraph contains args and kwarg, is it possible to return dict as result of inference model? 
It will be more align with OV backend.